### PR TITLE
add implementation

### PR DESCRIPTION
**Pull request by rraallvv**

### CODE REVIEW

File: NimicClientTest/Fixtures.cs [hunk 1]
```diff
@@ -0,0 +1,843 @@
+﻿using System;
+namespace NimiqClientTest
+{
+    public class Fixtures
+    {
+        public Fixtures()
+        {
+        }
+
+        public static string Accounts()
```
Comment (svub) [inline NimicClientTest/Fixtures.cs]: Similar to the Python client, also here it's only JSON results, no classes or structures that help a user to create this objects. I think this makes it much harder to use, no?

File: NimicClientTest/NimicClientTest.csproj [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (svub) [inline NimicClientTest/NimicClientTest.csproj:1]: The name of the file and folder is wrong... or is this intended for C#?

File: NimiqClient/NimiqClient.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <PackageVersion>0.0.4</PackageVersion>
+    <Authors>Nimiq Community</Authors>
+    <Copyright>Copyright (c) 2020 Nimiq Community</Copyright>
+    <Owners>Nimiq Community</Owners>
+    <PackageProjectUrl>https://github.com/rraallvv/NimiqClientSharp</PackageProjectUrl>
+    <Summary>https://github.com/rraallvv/NimiqClientSharp/blob/master/README.md</Summary>
+    <PackageTags>Nimiq, JSON-RPC Client</PackageTags>
```
Comment (svub) [inline NimiqClient/NimiqClient.csproj:11]: If these files are needed, then the URLs should be updated.

File: NimicClientTest/NimicClientTest.csproj (hunk 2 above)
Comment (svub) [reply NimicClientTest/NimicClientTest.csproj:1]: Ok, I think it's intended haha, but it looks a bit like a typo to me.

File: NimicClientTest/Fixtures.cs (hunk 1 above)
Comment (svub) [reply NimicClientTest/Fixtures.cs]: I see the objects exist, done

File: NimiqClientProjects.sln [hunk 4]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (svub) [inline NimiqClientProjects.sln:1]: neccessary file?

File: README.md [hunk 5]
```diff
@@ -1,26 +1,73 @@
...
-This implementation was originally contributed by [*Placeholder; Your name with link to GitHub*](https://github.com/nimiq/).
+This implementation was originally contributed by [rraallvv](https://github.com/rraallvv/).
 
 Please send your contributions as pull requests.
+
 Refer to the [issue tracker](https://github.com/nimiq-community/csharp-client/issues) for ideas.
 
+### Develop
+
+After cloning the repository, open the solucion file in the repository root folder.
```
Comment (svub) [inline README.md]: can not fine a "solucion" file.

File: ShowCase/Program.cs [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+            Console.WriteLine(await client.Consensus());
+
+            var accounts = await client.Accounts();
+
+            Console.WriteLine(accounts.Length);
+
+            foreach(var account in accounts)
+            {
+                Console.WriteLine(account);
+            }
```
Comment (svub) [inline ShowCase/Program.cs]: Would be cool if the examples can be a bit more extensive to really give a good starting point in using the cleint. Something more realistic and useful. Similar for other client libs.

File: ShowCase/ShowCase.csproj [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (svub) [inline ShowCase/ShowCase.csproj:1]: necessary file?

**Review by svub — CHANGES_REQUESTED** [review]
Would be great to have a more real world / extensive / useful example for new devs to get started. :)

Otherwise, looking good, maybe just check file and folder naming, make sure no files are checked in that are not needed (e.g. IDE settings)

File: NimicClientTest/Fixtures.cs (hunk 1 above)
Comment (rraallvv) [reply NimicClientTest/Fixtures.cs]: Yes, those json strings are used only in the tests, the are already predefined data structures used by all methods like in the recent changes made for the Python implementation.

File: NimicClientTest/NimicClientTest.csproj (hunk 2 above)
Comment (rraallvv) [reply NimicClientTest/NimicClientTest.csproj:1]: Yes, it's a C# implementation for the .Net [family of programming languages](https://en.wikipedia.org/wiki/List_of_CLI_languages). The client only depends on the package `System.Text.Json` which is available for [.Net Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-overview) and should be supported for [all these platforms](https://dotnet.microsoft.com/platform/dotnet-standard).

File: NimiqClient/NimiqClient.csproj (hunk 3 above)
Comment (rraallvv) [reply NimiqClient/NimiqClient.csproj:11]: It's been [fixed here](https://github.com/rraallvv/csharp-client/commit/03108cca7d1cd868192c0e1b205d0f8107655cb5)

File: NimiqClientProjects.sln (hunk 4 above)
Comment (rraallvv) [reply NimiqClientProjects.sln:1]: Yes, it's the Visual Studio solution file containing all the projects necessary to run the tests, run the showcase example, and pack the .Net Standard library (aka Nuget package).

File: README.md (hunk 5 above)
Comment (rraallvv) [reply README.md]: [Added](https://github.com/rraallvv/csharp-client/commit/4f28379fc27cfe940665f7368e831f6a28de659e) the name of the solution file to the README.md file

File: ShowCase/Program.cs (hunk 6 above)
Comment (rraallvv) [reply ShowCase/Program.cs]: [Improved](https://github.com/rraallvv/csharp-client/blob/master/ShowCase/Program.cs) the example program

File: ShowCase/ShowCase.csproj (hunk 7 above)
Comment (rraallvv) [reply ShowCase/ShowCase.csproj:1]: Yes, it's the Visual Studio project file used to run the showcase example.

**Comment by mariofriz** [comment]
It's probably a good PR but I don't even want to look at those 300+ files.
I know most of them are docs (because it's a full website), and makes me wonder if the website format is right for github.

**Comment by rraallvv** [comment]
@mariofriz , you'r right, the bulk of files is because of the automatically generated docs. I've been thinking about removing the docs all together and instead adding a section in the readme showing how to build and upload the Wiki to GitHub.

**Comment by rraallvv** [comment]
@mariofriz , the automatically generated wiki is available at https://github.com/rraallvv/csharp-client/wiki

The steps to generate the Markdown file that was uploaded to GitHub are here https://github.com/rraallvv/csharp-client#documentation

Now the PR has only 29 files in it. I'm going to do apply the same to the other client libraries.

**Comment by rraallvv** [comment]
I've put back the docs folder because it seems some of the other implementations are tracking the automatically generated docs, so now instead of a separate Wiki repository it's know accesible as a [static web page](http://rraallvv.github.io/csharp-client/) from the readme file. I've made the same changes to the Python and Swift implementations.

**Review by svub — APPROVED** [review]

**Pull request merged**